src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Scanner;
+
 public class Duke {
+    static WordList wordList;
+
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        wordList = new WordList();
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        replyWelcomeMessage();
+        String input;
+        while (true) {
+            input = sc.nextLine();
+            if (input.isEmpty()) {
+                warnEmpty();
+                continue;
+            }
+
+            Object[] parseResult = InputParser.parseInput(input);
+            InputType inputType = (InputType) parseResult[0];
+            String[] value = (String[]) parseResult[1];
+
+            processInput(inputType, value);
+            if (inputType == InputType.BYE) {
+                break;
+            }
+        }
+    }
+
+    public static void processInput(InputType inputType, String[] value) {
+        switch(inputType) {
+            case LIST:
+                wordList.printList();
+                break;
+            case MARK:
+                wordList.markItem(Integer.parseInt(value[0]));
+                break;
+            case UNMARK:
+                wordList.unmarkItem(Integer.parseInt(value[0]));
+                break;
+            case TODO:
+                wordList.storeTodo(value[0]);
+                break;
+            case DEADLINE:
+                wordList.storeDeadline(value[0], value[1]);
+                break;
+            case EVENT:
+                wordList.storeEvent(value[0], value[1]);
+                break;
+            case BYE:
+                replyBye();
+                break;
+            case NONE:
+                break;
+        }
+    }
```

Comment:
The coding standard for this module requires no indentation for the `case` clauses under `switch`! Perhaps you can configure your IDE to follow this new standard by default.